### PR TITLE
disable user-access to auto-updating results

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request removes user-access to auto-updating plugin results for the 3.10 release (will be used internally for #2827 and may or may not be exposed to the user in the future).

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone. Bugfix milestone also needs an accompanying backport label.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 New Features
 ------------
 
-- Infrastructure to support auto-updating plugin results. [#2680]
+- Infrastructure to support auto-updating plugin results, not yet available to users. [#2680, #2834]
```
Comment (kecnry) [inline CHANGES.rst:7]: we could instead just remove this entry altogether - any preferences?

Comment (pllim) [reply CHANGES.rst:7]: 👍 to remove change log entry. If user cannot use, it does not need an entry. Thanks!

File: jdaviz/components/plugin_add_results.vue [hunk 2]
```diff
@@ -52,7 +52,7 @@
 
     <slot></slot>
 
-    <v-row v-if="auto_update_result !== undefined">
+    <v-row v-if="false && auto_update_result !== undefined">
```
Comment (pllim) [inline jdaviz/components/plugin_add_results.vue:55]: Is `&&` even needed here?

Comment (kecnry) [reply jdaviz/components/plugin_add_results.vue:55]: as opposed to just having `false` (or removing it or commenting the whole block out)?  Any of those are options, sure.

Comment (kecnry) [reply jdaviz/components/plugin_add_results.vue:55]: How about I just remove the block and we can always revert from this diff when we want to reintroduce it?

Comment (pllim) [reply jdaviz/components/plugin_add_results.vue:55]: If you are going to reintroduce it soon, I'd say go for the easiest route but leave a comment. I'll leave the final choice to you. Thanks!

Comment (kecnry) [reply jdaviz/components/plugin_add_results.vue]: see latest commit for commenting out this block (and removing the `false`)

**Review by pllim — APPROVED** [review]
LGTM now. Thanks!

**Pull request merged**